### ISSUE TITLE
Automated Travis/AppVeyor integrations support? to build MacOS & windows versions

### ISSUE THREAD

**Issue by ahjolinna** [cmdrkotori/mpc-qt#169]
an app called [Orion](https://github.com/alamminsalo/orion) which is made using Qt5/qml and mpv just implemented travis/appveyor support for automatic win/mac build support, which made made me thinking if if you could add this to your project as this would remove some extra busy work and you would have an mac version as well.

if you are interested here is the Orion's [Travis](https://github.com/alamminsalo/orion/pull/182) support and [AppVeyor](https://github.com/alamminsalo/orion/pull/181)

**Comment by ahjolinna** [comment]
@cmdrkotori : do you think you could implement this? I personally think it would be great as you could also expand your user-base to MAC at the same time

**Comment by cmdrkotori** [comment]
Yes it certainly is possible. However I would first like to actually run this on a mac or a virtual machine to iron out the crinkles. I don't like coding blind.

Alternatively, if someone is willing to iron out the integration features, they're welcome to take this on before I get around to it.

**Comment by ahjolinna** [comment]
well you could at least start with the windows/AppVeyor, and why not still make mac/Travis version and just make it unofficial until there has been more proper testing done for it...but at least you will have the the building system implemented.

btw. here is small guide on how to get mac running on VM: [link](http://www.instructables.com/id/How-to-Run-Mac-OS-X-Inside-Windows-Using-VirtualBo/)

**Comment by cmdrkotori** [comment]
Thank you very much for the link. I tried months ago but had issues.

**Comment by albertosottile** [comment]
I just opened PR #237 with the goal to add Travis CI integration to generate distributable binaries for macOS. 

Speaking of ironing out some integration features, I also opened issue #236 that points exactly towards that goal.
